Add events.win destination docs (#6485)
segmentio/segment-docs: src/connections/destinations/catalog/events-win/index.md

---
title: events.win Destination
beta: true
id: 662d3328d029f89724a0c294
---

[events.win](https://events.win/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners){:target="\_blank”} provides a single platform to create your tracking plan, sync event definitions to your code, and see detailed metrics on how correct your data is. With events.win, you can ensure that your tracking is accurate and up-to-date.

This destination is maintained by events.win. For any issues with the destination, [contact the events.win support team](mailto:hi@events.win).

## Getting started

> 1. From your workspace's [Destination catalog page](https://app.segment.com/goto-my-workspace/destinations/catalog){:target="\_blank”} search for "events.win"
> 2. Select events.win and click **Add Destination**
> 3. Select an existing Source to connect to events.win (Actions).
> 4. Go to the [events.win dashboard](https://app.events.win/developers){:target="\_blank"}, find and copy the **Developer key**.
> 5. Enter the **Developer Key** in the events.win destination settings in Segment.
> 6. That's it! events.win will start to receive data from Segment, there may be a delay before data is visible in the events.win dashboard.

Reviewer: hi @megancooper - do you have a feel for how long the maximum delay would be? 

```suggestion
1. From your workspace's [Destination catalog page](https://app.segment.com/goto-my-workspace/destinations/catalog){:target="\_blank”} search for "events.win".
2. Select events.win and click **Add Destination**.
3. Select an existing Source to connect to events.win.
4. Go to the [events.win dashboard](https://app.events.win/developers){:target="\_blank"}, find and copy the **Developer key**.
5. Return to Segment and enter the **Developer Key** in the events.win destination settings.
6. events.win starts to receive data from Segment. There may be a delay before data is visible in the events.win dashboard.
```

Author: The delay at most would be a few minutes. I can add that detail in

## Supported methods

> events.win supports the following methods, as specified in the [Segment Spec](/docs/connections/spec).

Reviewer: ```suggestion
events.win supports Segment's [Track](/docs/connections/spec/track) method, as specified in the [Segment Spec](/docs/connections/spec).
```

### Track

> [Track](/docs/connections/spec/track) calls are consumed and validated against the tracking plan you've defined in events.win. We don't store the data, but we do provide a detailed report on how correct your data is.

Reviewer: ```suggestion
events.win consumes and validates [Track](/docs/connections/spec/track) calls against the tracking plan you've previously defined in events.win. events.win doesn't store the data, but instead provides a detailed report on how correct your data is.
```

> You can use the [@events.win/cli](https://www.npmjs.com/package/@events.win/cli) to generate type-safe tracking code for your events.

Reviewer: ```suggestion
You can use the [@events.win/cli](https://www.npmjs.com/package/@events.win/cli){:target="\_blank”} to generate type-safe tracking code for your events.
```
